src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -126,6 +126,28 @@ public function findForDetailsPage($id)
     /**
      * {@inheritdoc}
      */
+    public function findOneForPayment($id)
+    {
+        $queryBuilder = $this->createQueryBuilder('o');
```

Comment:
It is needed to create $queryBuilder? Cannot it be `return $his->createQueryBuilder('o')..`?